src/Areas/ContainerApps/Models/ContainerAppConfiguration.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+
+namespace AzureMcp.Areas.ContainerApps.Models;
+
+public class ContainerAppConfiguration
+{
+    [JsonPropertyName("ingress")]
+    public ContainerAppIngress? Ingress { get; set; }
+
+    [JsonPropertyName("secrets")]
```

Comment:
This comment is not related to the code itself 

Can the mcp server support [Ellicitation ](https://modelcontextprotocol.io/docs/concepts/elicitation)for secrets? That way based on user consent/awareness we display secret if the user consents to it?
Granted that not all clients support Ellicitation right now - VSCode does we can ask if VS will be doing so.

CC: @wbreza